.github/release-drafter.yml

```diff
@@ -47,9 +47,10 @@ autolabeler:
       - "**/*.md"
   - label: "enhancement"
```

Comment:
not sure *.go should be an enhancement.

Reply:
Since this was sitting on `internal/**/*` I figured I'd keep that semantic by making any Golang change an enhancement. I'm not too fuzzed about this anyway though.

Got an alternative proposal?

Reply:
I went and dropped the filter. I guess PRs should be manually labeled to be enhancement vs. bug fix.